index.html

```diff
@@ -442,19 +442,26 @@ <h3>The <dfn>PerformanceResourceTiming</dfn> Interface</h3>
 <li><dfn>"other"</dfn>, if none of the above conditions match.</li>
 </ul>
 <p data-dfn-for="PerformanceResourceTiming">On getting, the
-attribute <dfn>nextHopProtocol</dfn> returns the network protocol
-used to fetch the resource, as identified by the ALPN Protocol ID
-[[RFC7301]]; resources retrieved from <a data-cite=
-"HTML#relevant-application-cache">relevant application caches</a>
-or local resources, return an empty string. When a proxy is
-configured, if a tunnel connection is established then this
-attribute MUST return the ALPN Protocol ID of the tunneled
-protocol, otherwise it MUST return the ALPN Protocol ID of the
-first hop to the proxy. In order to have precisely one way to
-represent any ALPN protocol ID, the following additional
-constraints apply: octets in the ALPN protocol MUST NOT be
-percent-encoded if they are valid token characters except "%", and
-when using percent-encoding, uppercase hex digits MUST be used.</p>
+attribute <dfn>nextHopProtocol</dfn> returns the following:
+<ul>
+  <li>If the resource fails the <a>timing allow check</a> algorithm and the
+   resource <a>Request</a>'s <a
```

Comment:
So we need to get rid of this check about the destination in order to make this fully TAO protected, right?

Reply:
Yeah. Alternatively, we can wait for @noamr's Fetch Integration PR to land and fix this.

Reply:
The FETCH integration PR has landed, maybe this can be closed.